src/features/transfer/useTokenTransfer.ts

```diff
@@ -83,16 +84,20 @@ export function useTokenTransfer(onDone?: () => void) {
         const hypErc20 = isNativeToRemote
           ? getHypErc20CollateralContract(tokenRoute.hypCollateralAddress, provider)
           : getHypErc20Contract(tokenRoute.sourceTokenAddress, provider);
+        const gasPayment = await hypErc20.quoteGasPayment(destinationChainId);
```

Comment:
Flagging here that this method does not exist on the audited method, so we will probably want to make another PR for that version and then tag that as the audit-compliant version of the UI